sdks/java/io/jms/src/test/java/org/apache/beam/sdk/io/jms/BytesMessageToStringMessageMapper.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.sdk.io.jms;
+
+import javax.jms.BytesMessage;
+import javax.jms.Message;
+
+/**
+ * A test class that maps a {@link javax.jms.BytesMessage} into a {@link String}.
+ */
+public class BytesMessageToStringMessageMapper implements JmsIO.MessageMapper<String> {
```

Comment:
As this class is only used in the test, I think it's better to be an inner class of `JmsIOTest`.

Reply:
Thoughts ?

Reply:
Sounds good :)